tool/tctl/common/tctl.go

```diff
@@ -143,10 +148,12 @@ func TryRun(commands []CLICommand, args []string) error {
 		"Base64 encoded configuration string").Hidden().Envar(defaults.ConfigEnvar).StringVar(&ccf.ConfigString)
 	app.Flag("auth-server",
 		fmt.Sprintf("Attempts to connect to specific auth/proxy address(es) instead of local auth [%v]", defaults.AuthConnectAddr().Addr)).
+		Envar(proxyAddrEnvVar).
 		StringsVar(&ccf.AuthServerAddr)
```

Comment:
:nail_care: 
I assume it works but having Proxy Server and Auth Server in the same setting is confusing.

Reply:
It is confusing... 

I could rewrite this so that both `TELEPORT_PROXY` and `TELEPORT_AUTH_SERVER` are popped into this flag. Unfortunately, our CLI parsing library doesn't support multiple env vars, so this would have to be a bit of discrete logic. It'd be neat at some point if we introduced a `--proxy` flag to `tctl`, but until then, does this sound like a better compromise ?

Reply:
Yeah :+1: